security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_flower,flower,model_flower,base.group_user,1,1,1,1
\ No newline at end of file
+access_flower,flower,model_flower,base.group_user,1,1,1,1
+access_flower_water,flower_water,model_flower_water,base.group_user,1,1,1,1
```

Comment:
This is to limit who can create flower.water records, and thus limit who can water the plants. It would be great if you could also set the group on the button itself to limit its visibility and improve UX